dist/sbuttons.css

```diff
@@ -732,6 +732,24 @@ a.yellow-btn:active:not(.fill-color-btn) {
   height: 1.6em;
   padding: 0;
 }
+.add-to-cart-btn {
+  position: relative;
+  padding-right: 1.5em;
+}
+.add-to-cart-btn:after {
+  content: "\f7b6";
```

Comment:
The icon of the button is a mug which I think is a mistake, please change the code to the icon of a shopping cart which is "\f07a"

Reply:
I am changing it but after running npm run compile it is automatically changing to "\f7b6".

Reply:
Oh sorry, I meant to change it in the less file not in the css. Please do that then compile.

Reply:
Okay

Reply:
![2](https://user-images.githubusercontent.com/58396306/94339547-4ca48e80-0018-11eb-8a96-c39dfe6cfe15.JPG)
I don't know why I am getting this error while pushing.

Reply:
I google this error and it says I need to do force pushing should I?

Reply:
Done that. Thank You!
Now can you please check it.

Reply:
Can you please check it. I did that and I can see the desired changes.